<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmSplitControlForm</class>
 <widget class="QWidget" name="FlyEmSplitControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>643</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="Line" name="line_4">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font color=&quot;Green&quot;&gt;Data&lt;/font&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="menuPushButton">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveSeedPushButton">
         <property name="toolTip">
          <string>Save current seeds</string>
         </property>
         <property name="text">
          <string>Save Seeds</string>
         </property>
         <property name="icon">
          <iconset resource="../gui.qrc">
           <normaloff>:/images/save_seed.png</normaloff>:/images/save_seed.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="previewPushButton">
         <property name="text">
          <string>Preview Results</string>
         </property>
         <property name="icon">
          <iconset resource="../gui.qrc">
           <normaloff>:/images/swcpreview.png</normaloff>:/images/swcpreview.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="commitPushButton">
         <property name="toolTip">
          <string>Save splitting results into DVID</string>
         </property>
         <property name="text">
          <string>Save Splits</string>
         </property>
         <property name="icon">
          <iconset resource="../gui.qrc">
           <normaloff>:/images/save_split.png</normaloff>:/images/save_split.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitPushButton">
         <property name="toolTip">
          <string>Exit split and back to the proofreading mode</string>
         </property>
         <property name="text">
          <string>Exit Split</string>
         </property>
         <property name="icon">
          <iconset resource="../gui.qrc">
           <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="infoWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Line" name="line_5">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font color=&quot;Green&quot;&gt;View&lt;/font&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>3D Body Visualization</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="quickViewPushButton">
          <property name="toolTip">
           <string>3D body visualization (high resolution ball surface)</string>
          </property>
          <property name="text">
           <string>Ball Cloud (H)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coarseBodyViewPushButton">
          <property name="toolTip">
           <string>3D body visualization (low resolution ball surface)</string>
          </property>
          <property name="text">
           <string>Ball Cloud (L)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="meshPushButton">
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewResultQuickPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>3D split visualization</string>
        </property>
        <property name="text">
         <string>Splits</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="volumeViewLayout">
     <item>
      <widget class="QPushButton" name="viewSplitPushButton">
       <property name="text">
        <string>View 3D Split</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view3dBodyPushButton">
       <property name="text">
        <string>Volume View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="ZFlyEmBookmarkWidget" name="bookmarkWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmBookmarkWidget</class>
   <extends>QWidget</extends>
   <header location="global">flyem/widgets/zflyembookmarkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
